zgw_consumers/forms.py

```diff
@@ -0,0 +1,28 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+from OpenSSL import crypto
+
+from .models import Certificate
+
+
+class CertificateAdminForm(forms.ModelForm):
```

Comment:
if a private key is provided, shouldn't we validate that private key and public cert belong together?

edit: this seems to happen in the `models.py` already :)